dockerfiles/README.rst

```diff
@@ -33,6 +33,7 @@ with the included script
 
 .. code-block:: bash
 
+   $ pip install --upgrade pip setuptools setuptools_scm wheel
```

Comment:
Shouldn't this be covered by the build-system.requires property in pyproject.toml?

Reply:
For `pip`, see above. `setuptools_scm` is only installed as a build requirement, which won't cover `VERSION="$(python -m setuptools_scm)"` in `dockerfiles/build.sh` - so this is should probably be added to the `docker` extra, right? `setuptools` and `wheel` can probably be dropped here.

Reply:
Dropped `setuptools` and `wheel` in the latest PR revision.